### PR TITLE
doc: updating README

### PR DESCRIPTION
**Pull request by dwightmulcahy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,78 @@
...
+that will involve human interaction (me).
+
+# Installation
+_**THIS CURRENTLY ISN"T AVAILABLE YET!!!**_
+
+You can install `HealthCheck.Server` using pip:
+```
+pip install healthcheck.server
+```
+Currently it supports python versions 3.7+.
```
Comment (PapaMarky) [inline README.md]: Currently it _REQUIRES_

or "uses" or "is based on"

Comment (dwightmulcahy) [reply README.md]: done.

File: README.md [hunk 2]
```diff
@@ -47,21 +75,30 @@ if __name__ == "__main__":
...
+(https://tools.ietf.org/id/draft-inadarei-api-health-check-02.html#rfc.section.3) that allows the client to returns JSON 
+data that is stored in the health check log.
+
 ## healthchecker.server configuration
 
-HealthChecker.Server can be configured via command-line, environment variables or configuration file.
-Specifying command-line or environment options will override the configuration file options. Configuration 
-file options override default options. A combination of command-line, environment variables and configuration
-file can be used.
+HealthChecker.Server can optionally be configured via command-line, environment variables, or configuration file. Specifying command-line or environment options will override the configuration file options. Configuration file options override default options. A combination of command-line, environment variables, and the configuration file can be used for configuring the server.
```
Comment (dwightmulcahy) [inline README.md]: Ugh.  Don't like the way this flows...

File: README.md [hunk 3]
```diff
@@ -70,21 +107,22 @@ Test mode used for development.
...
 
 #### -ba, --bind_addr TEXT
 #### -p, --port INTEGER
 
 #### --config FILE
 Read configuration from FILE.  FILE defaults to `./config`. 
 
-Config file supports files formatted according to Configobj’s unrepr mode (https://configobj.readthedocs.io/en/latest/configobj.html#unrepr-mode).
+Read the configuration from FILE. FILE defaults to ./config.
+Config file supports files formatted according to Configobj's unrepr mode (https://configobj.readthedocs.io/en/latest/configobj.html#unrepr-mode).
```
Comment (dwightmulcahy) [inline README.md]: `unrepr-mode`

**Comment by dwightmulcahy** [comment]
Hmmm... not happy with the order/layout right now.  The next commit will address more of that.  For now, this WIP is getting this to at least somewhat of a point to work on.  :/

**Review by PapaMarky — COMMENTED** [review]
lgtm

**Pull request merged**